variants/models/scores.py

```diff
@@ -544,16 +545,28 @@ def unroll_extra_annos_result(rows, fields):
     return rows_
 
 
-def prioritize_genes(entrez_ids, hpo_terms, prio_algorithm):
+def prioritize_genes(job, entrez_ids, hpo_terms, prio_algorithm):
     """Perform gene prioritization query.
 
     Yield quadruples (gene id, gene symbol, score, priority type) for the given gene list and query settings.
```

Comment:
Adapt comment accordingly to reflect parameters.

Reply:
Done.